### PR TITLE
Add Gitlab CI template for the diagram cache

### PR DESCRIPTION
**Pull request by MoritzWeber0**

### CODE REVIEW

File: ci-templates/gitlab/README.md [hunk 1]
```diff
@@ -35,4 +36,28 @@ Make sure to enable the "Expand variable reference" flag.
...
-please refer to the [Gitlab CI template](./exporter.yaml).
+please refer to the [Gitlab CI template](./exporter.yml).
+
+## Diagram cache
+
+Please add the following section to your `.gitlab-ci.yml`:
+
+```yml
+variables:
+  CAPELLA_VERSION: 6.0.0 # Enter the Capella version of the model here, only versions >= 6.0.0 are supported
```
Comment (dominik003) [inline ci-templates/gitlab/README.md:47]: Is it correct that only 6.0.0 models are supported?

File: ci-templates/gitlab/exporter.yml [hunk 2]
```diff
@@ -15,7 +23,7 @@ export-to-t4c:
     FILE_HANDLER: local
 
     # Semantic Capella version (x.x.x) >= 6.0.0
-    # Defaults to ${CAPELLA_VERSION}-${CAPELLA_DOCKER_IMAGES_REVISION} if not defined
+    # Defaults to 6.0.0 if not defined
     CAPELLA_VERSION: 6.0.0
```
Comment (dominik003) [inline ci-templates/gitlab/exporter.yml:27]: In the `diagram-cache.yml` this line is commented out as it defaults to 6.0.0. I would either also comment it out here or not comment it out there.

**Review by dominik003 — APPROVED** [review]

**Review by dominik003 — APPROVED** [review]
Only two small comments, otherwise it looks good!

File: ci-templates/gitlab/README.md (hunk 1 above)
Comment (MoritzWeber0) [reply ci-templates/gitlab/README.md:47]: Good catch, I've removed it.

File: ci-templates/gitlab/exporter.yml (hunk 2 above)
Comment (MoritzWeber0) [reply ci-templates/gitlab/exporter.yml]: Done.

**Pull request merged**